routers/repo/pull.go

```diff
@@ -1162,6 +1162,22 @@ func DownloadPullDiffOrPatch(ctx *context.Context, patch bool) {
 
 	pr := issue.PullRequest
 
+	if pr.HasMerged {
+		diffType := git.RawDiffType("diff")
+		if patch {
+			diffType = git.RawDiffType("patch")
```

Comment:
These should use the constants.
https://github.com/go-gitea/gitea/blob/3723b0647f61717a249f2379b1fed1e6b4dbdbce/modules/git/diff.go#L24-L28

Reply:
@jolheiser done